tests/testthat/test-FilterState.R

```diff
@@ -155,7 +155,7 @@ testthat::test_that(
 
     testthat::expect_identical(
       shiny::isolate(filter_state$test_add_keep_na_call()),
-      quote(!is.na(test) & TRUE)
+      "!is.na(test) & TRUE"
```

Comment:
add_keep_na_call returns a call
```suggestion
      quote(!is.na(test) & TRUE)
```

Reply:
Test fixes are ready, pending discussion results.